README.md

```diff
@@ -1,65 +1,65 @@
-# Prometheus Exporter Snap for Hardware Observer charm
+# Prometheus Hardware Exporter
 
+Prometheus Hardware Exporter exports Prometheus metrics from BMCs (Baseboard
+management controllers) using IPMI (Intelligent Platform Management Interface)
+and Redfish protocol. It also exports and various SAS (Serial Attached SCSI) and
+RAID (redundant array of inexpensive disks) controllers.
 
-You can learn more about Hardware Observer on [Charmhub](https://charmhub.io/hardware-observer).
-## Getting Started
+Prometheus Hardware Exporter is recommended for [Juju](https://juju.is/) users,
```

Comment:
Non blocking. 
Another idea is that we don't need to mention the charm in exporter at all.

Reply:
I think this is fine to mention here, after all, this package is very hard to use without the charm to do some automation for you. And to avoid duplication of documentation effort, we need to reference to charmhub documentation